components/footers.js

```diff
@@ -29,6 +29,7 @@ export function Footer({ recentPost }) {
             <H5>About</H5>
             <FooterLink href="/about/">Overview</FooterLink>
             <FooterLink href="/team/">Team</FooterLink>
+            <FooterLink href="/presentations">Presentations & Press</FooterLink> 
```

Comment:
Guessing, but this probably needs to be: 


```suggestion
            <FooterLink href="/presentations">Presentations &amp; Press</FooterLink> 
```